src/pages/payments/routes.tsx

```diff
@@ -60,27 +60,11 @@ export const paymentRoutes = (
         />
       }
     >
+      <Route path="edit" element={<Edit />} />
       <Route path="documents" element={<Documents />} />
       <Route path="payment_fields" element={<PaymentFields />} />
       <Route path="apply" element={<Apply />} />
       <Route path="refund" element={<Refund />} />
     </Route>
-    <Route path=":id/edit" element={<Payment />}>
```

Comment:
@beganovich Is there any particular reason why we separated this route from the others? 

It looks like we have a free space to improve code clarity here. We had the same `guards` and the same `path` for both of them. With the older code we had unnecessary rendering every time we changed the tab to `Edit`. That's the actual reason for deleting this.

Reply:
Before it was, I think that case is gone.